plugin/src/main/java/com/denizenscript/denizen/objects/ChunkTag.java

```diff
@@ -338,6 +344,13 @@ public static void register() {
         // @returns ElementTag(Boolean)
         // @description
         // Returns true if the chunk has already been generated.
+        // @example
+        // # One way this can return "false" is if the player has been teleported to a location in the chunk while offline
+        // # to a chunk that has not been interacted or have not had a player nearby to generate it.
```

Comment:
If you have to explain the weird edge case where this example even applies, it's probably not a great example of the applicable tag. There are much more straightforward cases where this tag can be used.